WalletWasabi/Bases/ConfigBase.cs

```diff
@@ -64,7 +64,14 @@ public virtual async Task LoadOrCreateDefaultFileAsync()
 			}
 			else
 			{
-				await LoadFileAsync();
+				try
+				{
+					await LoadFileAsync();
+				}
+				catch(JsonReaderException)
+				{
+					Logger.LogInfo($"{GetType().Name} file is corrupted. Create default version at path: `{FilePath}`.");
```

Comment:
Is this better?
```suggestion
					Logger.LogInfo($"{GetType().Name} file has been deleted because it was corrupted. Recreated default version at path: `{FilePath}`.");
```